apps/frontend/README.md

```diff
@@ -12,6 +12,12 @@ Open [http://localhost:3000](http://localhost:3000) to view it in the browser.
 The page will reload if you make edits.<br />
 You will also see any lint errors in the console.
 
+#### Environment Variables
+
+The frontend application uses the production API by default, to change this behavior you can set the `REACT_APP_API_BASE_URL` environment variable.
```

Comment:
Does it? It seems like it defaults to `localhost`

Reply:
Last minute change that wasn't properly updated. Thanks